package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "inverted-index",
+  "version": "1.0.0",
+  "description": "Checkpoint project to create an inverted index",
+  "main": "index.js",
+  "scripts": {
+    "start": "gulp",
+    "test": "gulp test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/andela-mabdussalam/Inverted-index.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/andela-mabdussalam/Inverted-index/issues"
+  },
+  "homepage": "https://github.com/andela-mabdussalam/Inverted-index#readme",
+  "dependencies": {
+    "browser-sync": "^2.18.6",
+    "browserify": "^14.0.0",
+    "gulp": "^3.9.1",
+    "jasmine": "^2.5.3",
+    "jshint": "^2.9.4",
+    "gulp-browserify": "^0.5.1",
+    "gulp-rename": "^1.2.2",
+     "karma": "^1.4.1",
+    "karma-chrome-launcher": "^2.0.0",
+    "karma-coverage": "^1.1.1",
+    "karma-coveralls": "^1.1.2",
+    "karma-jasmine": "^1.1.0",
+    "karma-verbose-reporter": "0.0.5"
+  },
+  "devDependencies": {
+    "browserify": "^14.0.0",
+    "coveralls": "^2.11.15",
+    "eslint": "^3.13.1",
+    "eslint-config-airbnb-base": "^11.0.1",
+    "eslint-plugin-import": "^2.2.0",
+    "gulp-browserify": "^0.5.1",
+    "gulp-rename": "^1.2.2",
+    "karma": "^1.4.1",
+    "karma-chrome-launcher": "^2.0.0",
+    "karma-coverage": "^1.1.1",
+    "karma-coveralls": "^1.1.2",
+    "karma-jasmine": "^1.1.0",
+    "karma-verbose-reporter": "0.0.5"
+  }
+}
```

Comment:
You have multiple packages appearing simultaneously in your dependencies and dev-dependencies. They should appear in just one of them. If the package is not required to start your final app and is only used during development; it should reside in dev-dependencies.